recipes/containers/lighttpd.yaml

```diff
@@ -12,32 +12,58 @@ depends:
 
 
 buildScript: |
-    # start from scratch
-    rm -rf *
+    install_common() {
```

Comment:
No need to wrap it into a function if this is anyway invoked by all `multiPackage` entries.

Reply:
Well, actually I didn't know that... 😬